<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RollCallWindow</class>
 <widget class="QWidget" name="RollCallWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>随机点名</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,4,2,0,3,0">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelFontSize">
       <property name="text">
        <string>字体大小：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxFontSize"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelInterval">
       <property name="text">
        <string>间隔(ms)：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxInterval">
       <property name="minimum">
        <number>40</number>
       </property>
       <property name="maximum">
        <number>800</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string>未开始</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnStartStop">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="btnRemoveFromCalled">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="btnAddToCalled">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="4">
      <widget class="QListWidget" name="calledList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="4">
      <widget class="QListWidget" name="nameList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>待点名</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>已点名</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnReset">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnReset</sender>
   <signal>clicked()</signal>
   <receiver>RollCallWindow</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>898</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStartStop</sender>
   <signal>clicked(bool)</signal>
   <receiver>RollCallWindow</receiver>
   <slot>toggleStartStop(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>1190</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxFontSize</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>RollCallWindow</receiver>
   <slot>changeFontSize(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>1103</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStartStop</sender>
   <signal>clicked(bool)</signal>
   <receiver>btnAddToCalled</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStartStop</sender>
   <signal>clicked(bool)</signal>
   <receiver>btnRemoveFromCalled</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStartStop</sender>
   <signal>clicked(bool)</signal>
   <receiver>btnReset</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>557</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddToCalled</sender>
   <signal>clicked()</signal>
   <receiver>RollCallWindow</receiver>
   <slot>moveToAnother()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>834</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemoveFromCalled</sender>
   <signal>clicked()</signal>
   <receiver>RollCallWindow</receiver>
   <slot>moveToAnother()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>902</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>reset()</slot>
  <slot>toggleStartStop(bool)</slot>
  <slot>changeFontSize(int)</slot>
  <slot>moveToAnother()</slot>
 </slots>
</ui>
